### DISCUSSION TITLE
SK6812 WWA / RGBW / RGBWW / RGBNW

### DISCUSSION THREAD

**Discussion by andre2007** [onlaj/Piano-LED-Visualizer#503] [Q&A] [answered]
Hi,
on AliExpress I found a two meter 144 diodes LED strip, but I am unsure which variant will work with your project.

https://de.aliexpress.com/item/32664925043.html?spm=a2g0o.productlist.main.17.854002jY02jYUl&algo_pvid=918174ac-2e48-4db4-9bf4-3f07e4bcb578&algo_exp_id=918174ac-2e48-4db4-9bf4-3f07e4bcb578-8&pdp_npi=4%40dis%21EUR%2113.92%2110.44%21%21%2114.64%2110.98%21%40211b613117121721021321635ec417%2112000027094608799%21sea%21DE%210%21AB&curPageLogUid=PTk5ipOHbdzO&utparam-url=scene%3Asearch%7Cquery_from%3A

There are the variants: SK6812 WWA / RGBW / RGBWW / RGBNW

Which one will work, or will all work with your project?

Kind regards
André

**Comment by onlaj** [comment, marked as answer]
Hey.
None of them will work, you need WS2812B

**Reply by andre2007** [reply]
Thanks,  I found the correct one now.
